res/decoder-ring/decoder.js

```diff
@@ -117,7 +117,7 @@
             fetchStatus.fold({
             pending: ({ bytesReceived, length }) => {
                 if (!bytesReceived) {
-                    return e('span', { className: "spinner" }, "⦶");
+                    return e('span', { className: "spinner" }, "\u29b5");
```

Comment:
I think `<meta charset="utf-8">` would also have fixed this btw, assuming you're encoding your files as utf8.

Reply:
oh, whoops.